#
# Submitted-By: Luotao Fu, 2006-10-11
# Committed-By: Luotao Fu
#
# Error:
#
# Compilation fails if the bancomm interface is enabled in the configuration
#
# Description:
#
# wrong variable type of debug, the struct refclockproc contanins only nsec,
# instead of usec as used in the bancomm interface code.
#
# State:
#
# fixed, will be commited to the developer soon
#
# Reapplied for 4.2.4p3 - only some include line left, bug above fixed.
#

---
 ntpd/refclock_bancomm.c |    1 +
 1 file changed, 1 insertion(+)

Index: ntp-4.2.4p3/ntpd/refclock_bancomm.c
===================================================================
--- ntp-4.2.4p3.orig/ntpd/refclock_bancomm.c	2006-12-28 13:03:42.000000000 +0100
+++ ntp-4.2.4p3/ntpd/refclock_bancomm.c	2007-09-21 16:41:28.000000000 +0200
@@ -57,6 +57,7 @@
 #include <stdio.h>
 #include <syslog.h>
 #include <ctype.h>
+#include <sys/ioctl.h>
 
 /*  STUFF BY RES */
 struct btfp_time                /* Structure for reading 5 time words   */
